CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## New Features
 
+- PR #760 Raise `FileNotFoundError` instead of `GDF_FILE_ERROR` in `read_csv` if the file does not exist
```

Comment:
Changes look good, just missing a test. Something like:
with pytest.raises(FileNotFoundError):
        read_csv("random string")

with pytest.raises(FileNotFoundError):
        read_csv("nonexistingfilename.csv")
Would test both cases.

Reply:
Added a test to verify the FileNotFoundError, read_csv("non-existing-filename.csv").
I don't understand the difference between "random string" and "nonexistingfilename.csv", presumably both are likely to be non-existing filenames ?

Reply:
I misunderstood the isfile() check, sorry for the confusion. The intent was to have two cases that would fail the two different checks you added.